### PR TITLE
Replace ServoStackingContextId with ServoScrollRootId

### PR DESCRIPTION
**Pull request by mrobinson**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/467)

### CODE REVIEW

**Comment by mrobinson** [comment]
@glennw r?

File: webrender_traits/src/types.rs [hunk 1]
```diff
@@ -397,13 +397,13 @@ pub struct ScrollLayerId {
 #[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub enum ScrollLayerInfo {
     Fixed,
-    Scrollable(usize)
+    Scrollable(usize, ServoScrollRootId)
```
Comment (glennw) [inline webrender_traits/src/types.rs]: Should we just call this ScrollRootId instead?

**Comment by glennw** [comment]
@mrobinson OK, the WR update in master has landed. So feel free to merge this when you have the corresponding servo update ready.

File: webrender_traits/src/types.rs (hunk 1 above)
Comment (mrobinson) [inline webrender_traits/src/types.rs]: Hrm. I'm not sure. I kept the same naming scheme as ServoStackingContextId in the off-chance that WebRender also grew an understanding (and an id) for ScrollRoots. I guess we can land this PR and then investigate later once my first round of work is complete.

**Comment by mrobinson** [comment]
The Servo part of this change is here: https://github.com/mrobinson/servo/tree/scroll_id

**Comment by glennw** [comment]
@mrobinson The CI-related issues have been resolved, but this will need to be rebased then it should be fine to land.

**Comment by mrobinson** [comment]
@glennw Thanks. I'll try this one more time.

**Comment by mrobinson** [comment]
@emilio Thanks!

**Pull request merged**